src/modules/address-book/ModuleAddressBook.vue

```diff
@@ -334,8 +334,13 @@ export default {
     resolveAddress: throttle(async function () {
       if (this.nameResolver) {
         try {
+          // Ethers.js rejects Rootstock checksummed address so use lowercase address.
+          const inputAddress =
+            this.network.type.chainID === 30
```

Comment:
import and reference RSK type instead of hardcoding.

Reply:
Done ✅